### PR TITLE
Update item and holdings subfield names.

### PR DESCRIPTION
**Pull request by corylown**

### CODE REVIEW

File: lib/data/ncsu/traject_config.rb [hunk 1]
```diff
@@ -69,12 +69,12 @@ def item_status(current, home)
...
-    home = item['shelving_location']
+    current = item['loc_current']
+    home = item['loc_n']
     item['status'] = item_status(current, home)
 
-    if item.fetch('call_number_scheme', '') == 'LC'
-      lcc_top.add(item['call_number'][0, 1])
+    if item.fetch('cn_scheme', '') == 'LC'
+      lcc_top.add(item['call_no'][0, 1])
     end
```
Comment (kspurgin) [inline lib/data/ncsu/traject_config.rb]: Do we need item.delete('loc_current') before the next line?

File: lib/data/ncsu/traject_config.rb [hunk 2]
```diff
@@ -28,16 +28,16 @@
...
-  l: { key: 'shelving_location' },
+  i: { key: 'item_id' },
+  c: { key: 'copy_no' },
+  m: { key: 'loc_b' },
+  o: { key: 'notes' },
+  a: { key: 'call_no' },
+  k: { key: 'loc_current' },
+  l: { key: 'loc_n' },
   t: { key: 'type' },
   v: { key: 'volume' },
```
Comment (kspurgin) [inline lib/data/ncsu/traject_config.rb]: I'm adding 'volume' to Argot spec

File: lib/data/ncsu/traject_config.rb (hunk 1 above)
Comment (corylown) [reply lib/data/ncsu/traject_config.rb]: If loc_current isn't needed in the output then yes it can be deleted from the hash before it gets accumulated.

Comment (kspurgin) [reply lib/data/ncsu/traject_config.rb]: We hadn't defined a loc_current item key for Argot (OMG another kind of location!) so I was assuming it was just being used to determine status... @adjam can you confirm?

Comment (adjam) [reply lib/data/ncsu/traject_config.rb]: @corylown @kspurgin lcc_top can go away
loc_current is local to NCSU, it's how we compute availability.  don't touch it!

Comment (corylown) [reply lib/data/ncsu/traject_config.rb]: OK -- I think this can get merged as is then.

**Pull request merged**